### PR TITLE
Adds note on switching to OFI

### PR DESCRIPTION
**Pull request by aturner-epcc**

### CODE REVIEW

File: docs/research-software/vasp/vasp.md [hunk 1]
```diff
@@ -188,6 +188,26 @@ versions in the build instructions GitHub repository. See:
 
 ## Tips for using VASP on ARCHER2
 
+### Switching MPI transport protocol rom UCX to OpenFabrics
```
Comment (juanfrh) [inline docs/research-software/vasp/vasp.md:191]: "from" is misspelled. The rest looks good to me.

Comment (aturner-epcc) [reply docs/research-software/vasp/vasp.md]: Well spotted, fixed now

**Review by juanfrh — APPROVED** [review]
LGTM. Thanks Andy.

**Review by kevinstratford — COMMENTED** [review]
There is an implication here that with OFI, VASP does not pass all the regression tests. True?

If so, does a user switching to OFI need to know which tests don't pass?

**Comment by juanfrh** [comment]
Apologies, I approved the PR without reading Kevin's comment. I can revert the changes and create another PR. Please let me know if you want me to do so.

**Comment by aturner-epcc** [comment]
@kevinstratford That was not what I was trying to say (both OFI and UCX versions pass the same tests). Rather, I was trying to imply that both OFI and UCX pass the tests but UCX is better performing; however, this does not necessarily mean that the UCX version will never give errors and there is the potential that OFI will allow these cases to work.

**Pull request merged**